BUFRCREX_TableB_en_13.csv

```diff
@@ -75,3 +75,10 @@ ClassNo,ClassName_en,FXY,ElementName_en,BUFR_Unit,BUFR_Scale,BUFR_ReferenceValue
 13,Hydrographic and hydrological elements,013162,Cloud liquid water,kg m-2,2,0,8,kg m-2,2,3,,,Operational
 13,Hydrographic and hydrological elements,013163,Snow water equivalent,kg m-2,0,0,16,kg m-2,0,5,,,Operational
 13,Hydrographic and hydrological elements,013164,Sea ice freeboard,m,3,-131072,18,m,3,6,,,Operational
+13,Hydrographic and hydrological elements,013165,CAPE (convective available potential energy),J/kg,0,0,14,J/kg,0,5,(see Note 11),issue-162a,Operational
```

Comment:
@SibylleK Units don't match in the issue summary and branch. Should they be "J/kg" or "Jkg-1"?

Reply:
I would prefer J/kg.
The unit Joule per kilogram exists in the Common Code Tabel C-6 and the abbreviation in IA5/ASCII is J/kg. Therefore I took this different abbreviation for the new entries .